cdap-ui/app/cdap/components/AbstractWidget/AbstractMultiRowWidget/AbstractRow.tsx

```diff
@@ -82,22 +87,35 @@ export default class AbstractRow<
   };
 
   public render() {
+    const { errors, value } = this.props;
```

Comment:
nit: ```const { errors: IErrorObj, value: string } = this.props;```

Reply:
That would be aliasing syntax, right? Also, I specify the types in IAbstractRowProps earlier in the file.